src/mod.js

```diff
@@ -0,0 +1,15 @@
+export default mod
+
+/**
+ * Original Source: http://stackoverflow.com/a/17323608
+ *
+ * This method provides a modulo function that will always return a positive
+ * value even when supplied a negative dividend.
+ *
+ * @param {Date} date - date to be evaluated
+ * @returns {Boolean} - True if date is today, false otherwise
```

Comment:
I think this JSDoc is describing the params/returns incorrectly. Looks like copy/paste 😅